### PR TITLE
Practicak Work 9

### PR DESCRIPTION
**Pull request by Valeriy-Svetlichniy**

### CODE REVIEW

File: Svetlichniy_Valeriy/Practical Work № 9/Task № 1/Source.cpp [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	file << a.Name << "; " << a.LastName << "; " << a.E_mail << "; " << a.Password << endl;
+	file.close();
+}
+
+main() 
+{
+	setlocale(LC_ALL, "Russian");
+	User b;
+	double n;
+	cout << "������� ���������� ������������� ";
```
Comment (kotsolesya) [inline Svetlichniy_Valeriy/Practical Work № 9/Task № 1/Source.cpp]: only english

Comment (maridzhe) [reply Svetlichniy_Valeriy/Practical Work № 9/Task № 1/Source.cpp]: Rly,  why you not using english??

Comment (Valeriy-Svetlichniy) [reply Svetlichniy_Valeriy/Practical Work № 9/Task № 1/Source.cpp]: Because it's not so interesting!

**Comment by kotsolesya** [comment]
ок

**Pull request closed without merging**